python/ray/tune/suggest/bayesopt.py

```diff
@@ -76,6 +74,15 @@ def __init__(self,
             max_concurrent=max_concurrent,
             use_early_stopped_trials=use_early_stopped_trials)
 
+        if utility_kwargs is None:
+            # The defaults arguments are the same
+            # as in the package BayesianOptimization
+            utility_kwargs = dict(
+                kind="ucb",
+                kappa=2.576,
+                xi=0.0,
```

Comment:
Can you add a note in the docstring that the values are defaulted to this?

Reply:
Sure!